docs/source/docs/integration/simpleStrategies.md

```diff
@@ -1,36 +1,30 @@
 # Simple Strategies
 
-Simple strategies for using vision processor outputs involve using the target's position in the 2D image to infer *range* and *angle* to the target.
+Simple strategies for using vision processor outputs involve using the target's position in the 2D image to infer *range* and *angle* to a particular AprilTag.
 
 ## Knowledge and Equipment Needed
 
 - A Coprocessor running PhotonVision
 - A Drivetrain with wheels
+- An AprilTag to aim at
 
 ## Angle Alignment
 
-The simplest way to use a vision processing result is to first determine how far left or right in the image the vision target should be for your robot to be "aligned" to the target. Then,
+The simplest way to align a robot to an AprilTag is to rotate the drivetrain until the tag is centered in the camera image. To do this,
 
-1. Read the current angle to the target from the vision Coprocessor.
-2. If too far in one direction, command the drivetrain to rotate in the opposite direction to compensate.
+1. Read the yaw current angle to the AprilTag from the vision Coprocessor.
```

Comment:
current yaw angle not yaw current angle, add admonition either here or on aiming at a target on the classic "what happens if our camera isnt centered" (this doesnt have to turn to center of the frame, etc. etc.). Not sure of the most natural place for that information to exist, but it should because a student has asked me it more than once